go-controller/pkg/ovn/egressip.go

```diff
@@ -1643,7 +1646,7 @@ func (oc *Controller) deleteEgressNode(egressNode *kapi.Node) error {
 	}
 	err := libovsdbops.UpdateLogicalSwitchPortSetOptions(oc.nbClient, &lsp)
 	if err != nil {
-		klog.Errorf("Unable to remove GARP configuration on external logical switch port for egress node: %s, err: %v", egressNode.Name, err)
+		return fmt.Errorf("unable to remove GARP configuration on external logical switch port for egress node: %s, err: %v", egressNode.Name, err)
```

Comment:
interesting as when I simulated errors during ut by bringing down NBDB that was the error I was hitting and when NBDB is back we were recovered 
here is the error seen when forced NBDB down
```
E0506 11:49:39.483137    9711 obj_retry.go:1448] Failed to update resource *factory.egressNode, old=node2, new=node2, error: unable to configure GARP on external logical switch port for egress node: node2, this will result in packet drops during egress IP re-assignment,  err: error in transact with ops [{Op:update Table:Logical_Switch_Port Row:map[options:{GoMap:map[exclude-lb-vips-from-garp:true nat-addresses:router router-port:rtoe-GR_node2]}] Rows:[] Columns:[] Mutations:[] Timeout:<nil> Where:[where column _uuid == {30858b4e-6ce2-4434-a77a-fffd07ed62e0}] Until: Durable:<nil> Comment:<nil> Lock:<nil> UUIDName:}]: context deadline exceeded: while awaiting reconnection
E0506 11:49:39.486302    9711 obj_retry.go:1489] Failed to delete resource object node1 of type *factory.egressNode, error: unable to remove GARP configuration on external logical switch port for egress node: node1, err: error in transact with ops [{Op:update Table:Logical_Switch_Port Row:map[options:{GoMap:map[router-port:rtoe-GR_node1]}] Rows:[] Columns:[] Mutations:[] Timeout:<nil> Where:[where column _uuid == {10453c36-cfd3-41ad-be37-8672cbabc194}] Until: Durable:<nil> Comment:<nil> Lock:<nil> UUIDName:}]: context deadline exceeded: while awaiting reconnection
```